### ISSUE TITLE
 Show time remaining when syncing to/from remote devices 

### ISSUE THREAD

**Issue by jony0008** [syncthing/syncthing#3698]

**Comment by calmh** [comment]
Yep. We could do a rough estimate based on recent history, although it will be a rough guess only.

**Comment by jony0008** [comment]
Why not use the download/upload speed for calculating the time for syncing?

**Comment by calmh** [comment]
Of course we would do that. It's still only a rough estimate, because we don't know ahead of time how much data for a given file will need to be transferred vs recycled locally.
